wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -960,51 +960,6 @@ extension WireCallCenterV3 {
         return conversationType
     }
 
-    /// Set MLSConferenceInfo to AVSWrapper in case this is a MLS conference
-    func setMLSConferenceInfoIfNeeded(for conversationId: AVSIdentifier) {
-        Task {
-            guard let syncContext = await self.uiMOC?.perform({ self.uiMOC?.zm_sync }) else { return }
-
-            let result: (MLSServiceInterface?, QualifiedID?, MLSGroupID?) = await syncContext.perform {
-                let conversation = ZMConversation.fetch(
-                    with: conversationId.identifier,
-                    domain: conversationId.domain,
-                    in: syncContext
-                )
-                guard let conversation, conversation.avsConversationType == .mlsConference else {
-                    return (nil, nil, nil)
-                }
-                return (syncContext.mlsService,
-                        conversation.qualifiedID,
-                        conversation.mlsGroupID)
-            }
-
-            guard
-                let mlsService = result.0,
-                let parentQualifiedID = result.1,
-                let parentGroupID = result.2
-            else {
-                return
-            }
-
-            do {
-                let subgroupID = try await mlsService.createOrJoinSubgroup(
-                    parentQualifiedID: parentQualifiedID,
-                    parentID: parentGroupID
-                )
-
-                let conferenceInfo = try await mlsService.generateConferenceInfo(
-                    parentGroupID: parentGroupID,
-                    subconversationGroupID: subgroupID
-                )
-
-                self.avsWrapper.setMLSConferenceInfo(conversationId: conversationId, info: conferenceInfo)
-            } catch {
-                WireLogger.mls.error("error while setMLSConferenceInfo: \(error.localizedDescription)")
-            }
-        }
-    }
```

Comment:
why has this been removed?

Reply:
I think this is dead code. We don't use it. Do you have any concerns?

Reply:
Ok 👍 Looks fine to remove